segnet/utils/timing.py

```diff
@@ -0,0 +1,95 @@
+
+import os
+import json
+import datetime
+
+from functools import wraps
+from time import time
+from typing import Callable
+
+def time_this(f) -> Callable:
```

Comment:
It seems like you annotate most of the code, which is fine, so this should also be annotated?

Reply:
Right on! I added the annotations at the beginnig, then forgot to annotate the rest of the code.